docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.6'
+services: 
+    postgres:
+        image: postgres:15-alpine 
+        restart: always
+        environment: 
+            # - DATABASE_HOST=127.0.0.1
+            - POSTGRES_USER=root
+            - POSTGRES_PASSWORD=root
+            - POSTGRES_DB=auth_development
+        ports: 
+            - "5432:5432"
+        volumes:
+            - ./db/db_users_table.sql:/docker-entrypoint-initdb.d/db_users_table.sql
+    
+    pgadmin-compose:
+        image: dpage/pgadmin4
+        environment: 
+            PGADMIN_DEFAULT_EMAIL: "test@gmail.com"
+            PGADMIN_DEFAULT_PASSWORD: "test123!"
```

Comment:
let's make the password be the same between the DB and admin (for development purpose only)